tritium-caffeine/src/test/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheMetricsTest.java

```diff
@@ -74,16 +81,48 @@ void estimatedSize() {
     }
 
     @Test
-    void maximumSize() {
+    void unboundedMaximumSize() {
+        when(cache.policy().eviction()).thenReturn(Optional.empty());
         assertThat(metrics.get("test.cache.maximum.size"))
                 .isInstanceOf(Gauge.class)
                 .returns(-1L, metric -> ((Gauge<?>) metric).getValue());
```

Comment:
I debated whether to change `CaffeineStats::maximumSize()` behavior to return `0` when no maximum is set as that would simplify some metrics downstream if we filter zeros, but didn't want to make behavior change in an excavator bump of mockito test dependency.

https://github.com/palantir/tritium/blob/2895ab4ccdf1a7d5d689d5b806305761b063dc31/tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CaffeineStats.java#L64-L66